ledger/src/blockstore.rs

```diff
@@ -1569,20 +1575,14 @@ impl Blockstore {
     fn make_transaction_status_index(
         &self,
         index: (Signature, Slot),
+        w_active_transaction_status_index: RwLockWriteGuard<u64>,
```

Comment:
this can be a &mut u64, you can pass in a &mut to the `RwLockWriteGuard `to the function

Reply:
Oh, way better, thx!